.gitignore

```diff
@@ -5,6 +5,7 @@
 .#*
 *~
 *.swp
+.vscode
```

Comment:
I would prefer if this line will be in a separate patch (but keep it in this merge)
The main reason this is to keep main patch contains only prepare/check bits